packages/shared/src/components/marketingCta/MarketingCtaBriefing.tsx

```diff
@@ -0,0 +1,190 @@
+import React, { useCallback, useEffect, useRef } from 'react';
+import classNames from 'classnames';
+import { format } from 'date-fns';
+import type { MarketingCta } from './common';
+import {
+  Typography,
+  TypographyColor,
+  TypographyTag,
+  TypographyType,
+} from '../typography/Typography';
+import { Button, ButtonSize, ButtonVariant } from '../buttons/Button';
+import { briefButtonBg } from '../../styles/custom';
+import { ClickableCard } from '../cards/common/Card';
+import { useIsLightTheme } from '../../hooks/utils';
+import { IconSize } from '../Icon';
+import {
+  AnalyticsIcon,
+  BellDisabledIcon,
+  BriefIcon,
+  EyeIcon,
+  MenuIcon,
+  TimerIcon,
+} from '../icons';
+import {
+  DropdownMenu,
+  DropdownMenuContent,
+  DropdownMenuOptions,
+  DropdownMenuTrigger,
+} from '../dropdown/DropdownMenu';
+import { useActions, useBoot } from '../../hooks';
+import { ActionType } from '../../graphql/actions';
+import { LogEvent, TargetType } from '../../lib/log';
+import { useLogContext } from '../../contexts/LogContext';
+import { anchorDefaultRel } from '../../lib/strings';
+
+const stats = [
+  {
+    Icon: TimerIcon,
+    label: 'Save 15-20 hours of reading',
+  },
+  {
+    Icon: AnalyticsIcon,
+    label: 'Analyzed 5,000+ posts and videos',
+  },
+];
+
+export const MarketingCtaBriefing = ({
+  campaignId,
+  flags: { title, description, ctaText, ctaUrl },
+}: MarketingCta) => {
+  const { completeAction } = useActions();
+  const isLightMode = useIsLightTheme();
+  const { clearMarketingCta } = useBoot();
+  const { logEvent } = useLogContext();
+  const hasSentImpression = useRef(false);
+
+  const hideCard = useCallback(() => {
+    // log dismiss event
+    logEvent({
+      event_name: LogEvent.MarketingCtaDismiss,
+      target_type: TargetType.MarketingCtaBrief,
+      target_id: campaignId,
+    });
+
+    clearMarketingCta(campaignId);
+  }, [campaignId, clearMarketingCta, logEvent]);
+
+  const onNeverShowAgain = useCallback(() => {
+    hideCard();
+    completeAction(ActionType.DisableBriefCardCta);
+  }, [completeAction, hideCard]);
+
+  const onClickCta = useCallback(() => {
+    hideCard();
```

Comment:
This will trigger two events, clicking the CTA and the dismissal itself. Not sure if that is accurate.